configs/setups/awiesm/awiesm.yaml

```diff
@@ -80,13 +79,6 @@ general:
             fesom_Dflags: "-DFESOM_COUPLED=ON"
             with_recom: false
           '2.1-recom':
-            couplings:
-            - fesom-2.1-wiso+echam-6.3.05p2-wiso+recom-2.0
-            add_include_models:
-            - recom
-            fesom_Dflags: "-DFESOM_COUPLED=ON -DRECOM_COUPLED=ON"
-            with_recom: true
-          '2.1-recom-par-tracers':
             couplings:
             - fesom-2.1-recom-par_tracers_mpi+echam-6.3.05p2-wiso+recom-2.0
             add_include_models:
```

Comment:
Is `- recom` not needed any more (in add_include_models)? and the switch for recom `with_recom`?